airbyte-integrations/connectors/source-salesforce/unit_tests/test_memory.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+import tracemalloc
+
+import pytest
+import requests_mock
+from conftest import generate_stream
+from source_salesforce.streams import BulkIncrementalSalesforceStream
+
+
+@pytest.mark.parametrize(
+    "n_records, first_size, first_peak",
+    (
+        (1000, 0.4, 1),
+        (10000, 1, 2),
+        (100000, 4, 7),
+        # (200000, 6, 12), # takes a long time
```

Comment:
just to confirm - it takes a long time but it passes?

Reply:
yes)